tensorflow/lite/micro/tools/make/targets/cortex_m_corstone_300_makefile.inc

```diff
@@ -102,7 +91,57 @@ ifneq ($(filter cortex-m55%,$(TARGET_ARCH)),)
   endif
 endif
 
-ifeq ($(TOOLCHAIN), gcc)
+# Filter out part of mcpu string for choosing the correct startup files
+ARM_CPU := $(subst cortex-m,ARMCM,$(MCPU_OPTION))
+ARM_CPU := $(subst +nofp,,$(ARM_CPU))
+
+ifeq ($(TOOLCHAIN), armclang)
+  CXX_TOOL  := armclang
+  CC_TOOL   := armclang
+  AR_TOOL   := armar
+  LD        := armlink
+
+  FLAGS_ARMC = \
+    --target=arm-arm-none-eabi \
+    -Wno-unused-private-field \
+    -mcpu=$(MCPU_OPTION)
+
+  ARM_LDFLAGS += --strict --summary_stderr --info summarysizes --map \
+                 --load_addr_map_info --xref --callgraph --symbols --info sizes \
+                 --info totals --info unused --info veneers \
+                 --list=${TENSORFLOW_ROOT}$(MAKEFILE_DIR)/gen/$(TARGET).map \
+                 --entry=Reset_Handler --verbose \
+                 --scatter=$(ETHOS_U_CORE_PLATFORM)/platform.scatter
+
+  # Pass a hint to the linker where to find the entry point. This needs to be
+  # done since the startup object file (containing the entry point) is inside
+  # the TFLM library. See:
+  # https://developer.arm.com/documentation/ka003125/latest
+  ARM_LDFLAGS += $(LIBDIR)/$(MICROLITE_LIB_NAME)\(startup_$(ARM_CPU).o\)
+
+  # -Wl takes comma separated values, replace space with comma's
```

Comment:
can `ARM_LDFLAGS`` be a comma separated list to begin with?

Does this work?

```
ARM_LDFLAGS = $(ARM_LDFLAGS),$(LIBDIR)/$(MICROLITE_LIB_NAME)\(startup_$(ARM_CPU).o\)
```

Reply:
Yes, if you use `-Wl`. Checkout [this link](https://www.keil.com/support/man/docs/armclang_intro/armclang_intro_chr1382712068183.htm).